configs/fastfcn/fastfcn-jpu_deeplabv3_512x1024_80k_cityscapes.py

```diff
@@ -0,0 +1,19 @@
+# model settings
```

Comment:
`configs/fastfcn/fastfcn-jpu_deeplabv3_512x1024_80k_cityscapes.py`
-->
`configs/fastfcn/fastfcn_r50-d32_jpu_aspp_512x1024_80k_cityscapes.py`

Reply:
include the README.md, model path, and log path.